Source/Core/Common/MemArena.cpp

```diff
@@ -63,7 +63,7 @@ void MemArena::GrabSHMSegment(size_t size)
 #else
 	for (int i = 0; i < 10000; i++)
 	{
-		std::string file_name = StringFromFormat("dolphinmem.%d", i);
+		std::string file_name = StringFromFormat("/dolphinmem.%d", i);
```

Comment:
The SUS actually says it's "platform defined" if it doesn't start with a slash, and FreeBSD was retuning EAGAIN (which doesn't show up in the manpage...) but it works if I add the slash.